test/e2e_node/density_test.go

```diff
@@ -530,7 +530,8 @@ func createBatchPodSequential(f *framework.Framework, pods []*v1.Pod) (time.Dura
 	e2eLags := make([]e2emetrics.PodLatencyData, 0)
 	for _, pod := range pods {
 		create := metav1.Now()
-		f.PodClient().CreateSync(pod)
+		p := f.PodClient().Create(pod)
+		framework.ExpectNoError(e2epod.WaitTimeoutForPodReadyInNamespaceWithInterval(f.ClientSet, p.Name, p.Namespace, 500*time.Millisecond, framework.PodStartTimeout))
```

Comment:
why polling faster solves the problem?

Reply:
Sorry, I just see the test is using `LatencyMetric` which is not related with the checking interval.

It should not be related.

Reply:
then I think that the right fix is to use a listWatcher, same technique is used by sig-scalability

https://github.com/kubernetes/perf-tests/blob/1e3b84449e818ab9ae3a0336da53f374c6dea560/clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go#L127-L155

Reply:
I have changed to use informer which has been used by `runDensityBatchTest`, /cc @aojea @pacoxu